src/components/People.tsx

```diff
@@ -0,0 +1,88 @@
+import { useEffect, useState } from 'react';
+import { getPeople } from '../api';
+import { Person } from '../types';
+import { Loader } from './Loader';
+import { PersonLink } from './PersonLink';
+import { getPeopleWithParents } from '../utils/utils';
+
+export const People: React.FC = () => {
+  const [peopleFromServer, setPeopleFromServer] = useState<Person[] | null>(
+    null,
+  );
+  const [error, setError] = useState<string | null>(null);
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const isArr = Array.isArray(peopleFromServer);
```

Comment:
looks redundant right now

Reply:
if I remove null from the types, then the message will be displayed for the first 0.5 seconds on boot, as the state will be checked, for example (`!error && peopleFromServer.length === 0 && { ... }`)
only when the state is renewed, `no people` will disappear